cli/unregister/unregister.go

```diff
@@ -205,24 +206,40 @@ func DeepClean() error {
 		cliutils.RunCmd(nil, "pkill", "-f", "/usr/horizon/bin/anax")
 
 	} else {
-		cliutils.Verbose(msgPrinter.Sprintf("Stopping horizon..."))
-		cliutils.RunCmd(nil, "systemctl", "stop", "horizon.service")
-
-		msgPrinter.Printf("Deleting local horizon DB...")
-		msgPrinter.Println()
-		cliutils.RunCmd(nil, "bash", "-c", "rm -f /var/horizon/*.db")
-		cliutils.RunCmd(nil, "bash", "-c", "rm -Rf /etc/horizon/policy.d/*")
+		if runtime.GOOS == "darwin" {
+			outBytes, errBytes := cliutils.RunCmd(nil, "/bin/sh", "-c", "docker ps | grep horizon1")
+			if errBytes != nil && len(errBytes) > 0 {
+				return fmt.Errorf(msgPrinter.Sprintf("Error during checking if anax container is running: %s", errBytes))
+			}
+			if outBytes != nil && len(outBytes) > 0 {
+				cliutils.Verbose(msgPrinter.Sprintf("Restarting horizon container..."))
+				outBytes, errBytes = cliutils.RunCmd(nil, "/bin/sh", "-c", "/usr/local/bin/horizon-container update")
```

Comment:
'update' might not work here. This is 'deep clean' part, the user usually runs it when the unregistration failed or could not stop so they have to ctrl-c out and run this command with -D. You can add the commands `docker exec -it <horizonx>  rm -f /var/horizon/*.db` and `docker exec -it <horizonx>  rm -Rf /etc/horizon/policy.d/*` before this 'update' command.